.github/workflows/k8s_e2e.yaml

```diff
@@ -0,0 +1,69 @@
+name: Kubernetes e2e testing
+
+env:
+  EXTERNAL_REGISTRY: true
+  REGISTRY_PORT: 80
+
+on:
+  # push:
+  #   paths-ignore:
+  #     # - '.github/**'
+  #     - '.gitignore'
+  #     - 'LICENSE'
+  #     - 'README*'
+  #   branches: ['*']
+    # branches: [master]
+#TODO choose to fire up with cron or on push to master
+# on:
+#   schedule:
+#     - cron:  '0 */2 * * *'
+
+#This trigger is only for testing
+  pull_request:
```

Comment:
Do we want the workflow to act upon the feature branch from which the pull request has been raised to master?
If yes, we're good.
However, if we intend the workflow to act upon the master branch once the changes are merged to the master, we should change the trigger event to `push`

Reply:
no worries. I'll change the trigger before merging the PR, otherwise it wouldn't be able to test if the scripts work beforehand